tap_mssql/__init__.py

```diff
@@ -331,7 +331,7 @@ def desired_columns(selected, table_schema):
             not_selected_but_automatic,
         )
 
-    return selected.intersection(available).union(automatic)
+    return sorted(selected.intersection(available).union(automatic), key = list(table_schema.properties.keys()).index)
```

Comment:
Can `table_schema.properties.keys()` be of different order itself on different runs?

Reply:
No, it maintains the same order on different runs